tests/end-to-end/upgrade/README.md

```diff
@@ -55,9 +55,14 @@ go run main.go --action prepareData --verbose
 
 ## Development
 
+>**NOTE:** The following approach for adding new tests to the upgrade scenario is not needed anymore. To prepare data during the preparation phase just create a new testdefinition that executes your preparation code and add the label `kyma-project.io/before-upgrade=true`. To execute evaluation code during the evaluation phase create a new testdefinition with the label `kyma-project.io/after-upgrade=true` 
```

Comment:
```suggestion
```

Reply:
It seems like the exact same note is repeated 2 lines further. Maybe we only one of them would be enough. Since the note is about "adding new tests to the upgrade scenario", I think it should be under the "Add a new test" title.